benches/pub_sub.rs

```diff
@@ -86,8 +93,10 @@ impl Actor for SubscriberActor {
     fn handle(
         &mut self,
         _context: &mut Self::Context,
-        _message: Self::Message,
+        message: Self::Message,
     ) -> Result<(), Self::Error> {
+        // This black_box has a nice side effect that it silences the 'field is never read' warning.
+        black_box(message.0);
```

Comment:
This is part of the https://github.com/tonarino/actor/pull/87/commits/d3713d6db110ef91b590667a6c4a5987fa83d570 commit that intentionally changes the performance profile of the benchmark (and does nothing else). We take it as the new baseline.

It is here mainly to suppress the warning, but also to prevent an unlikely situation that this whole method is more or less compiled out.